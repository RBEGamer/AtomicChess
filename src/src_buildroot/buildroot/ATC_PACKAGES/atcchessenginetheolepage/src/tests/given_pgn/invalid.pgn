[Event "?"]
[Site "?"]
[Date "1904.??.??"]
[Round "?"]
[White "Giese"]
[Black "Alekhine, Alexander A"]
[Result "0-1"]
[ECO "C33"]

1. e2-e4 e7-e5 2. f2-f4 e5xf4 3. Bf1-c4 d7-d5 4. Bc4xd5 Qd8-h4+ 5. Ke1-f1
g7-g5 6. Nb1-c3 Ng8-e7 7. d2-d4 Bf8-g7 8. Ng1-f3 Qh4-h5 9. h2-h4 h7-h6 10.
e4-e5 Nb8-c6 11. Kf1-g1 g5-g4 12. Nf3-e1 Bc8-f5 13. Bd5xc6+ Ne7xc6 14.
Nc3-e2 Bf5-e4 15. Bc1xf4 Qh5-f5 16. Qd1-d2 O-O-O 17. Ne2-g3 Qf5-h7 18.
Qd2-e2 Nc6xd4 19. Qe2-c4 Be4-c6 20. c2-c3 Nd4-e6 21. Qc4-f1 h6-h5 22.
Bf4-g5 Bg7xe5 23. Bg5xd8 Be5xg3 24. Bd8-f6 Qh7-e4 25. Ne1-d3 Ne6-f4 26.
Rh1-h3 Qe4-e3+ 27. Ne8-a7 Qg3-g4 0-1

