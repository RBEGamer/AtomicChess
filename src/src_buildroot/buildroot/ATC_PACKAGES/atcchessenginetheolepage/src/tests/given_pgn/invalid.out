[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[piece_taken]	 piece: 5 file: 6 rank: 4
[move]	board::PieceType: 3
[move]	board::PieceType: 5
[move]	board::PieceType: 3
[piece_taken]	 piece: 5 file: 4 rank: 5
[move]	board::PieceType: 1
[check]	: color: 0
[move]	board::PieceType: 0
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 5
[move]	board::PieceType: 3
[move]	board::PieceType: 4
[move]	board::PieceType: 1
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 0
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 3
[move]	board::PieceType: 3
[piece_taken]	 piece: 4 file: 3 rank: 6
[check]	: color: 1
[move]	board::PieceType: 4
[piece_taken]	 piece: 3 file: 3 rank: 6
[move]	board::PieceType: 4
[move]	board::PieceType: 3
[move]	board::PieceType: 3
[piece_taken]	 piece: 5 file: 6 rank: 4
[move]	board::PieceType: 1
[move]	board::PieceType: 1
[move]	board::PieceType: 0
[queenside castling]	color: 1
[move]	board::PieceType: 4
[move]	board::PieceType: 1
[move]	board::PieceType: 1
[move]	board::PieceType: 4
[piece_taken]	 piece: 5 file: 4 rank: 4
[move]	board::PieceType: 1
[move]	board::PieceType: 3
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 1
[move]	board::PieceType: 5
[move]	board::PieceType: 3
[move]	board::PieceType: 3
[piece_taken]	 piece: 5 file: 5 rank: 5
[move]	board::PieceType: 3
[piece_taken]	 piece: 2 file: 4 rank: 8
[move]	board::PieceType: 3
[piece_taken]	 piece: 4 file: 7 rank: 3
[move]	board::PieceType: 3
[move]	board::PieceType: 1
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 2
[move]	board::PieceType: 1
[check]	: color: 0
[disqualified]	: color: 0
[game_finished]	
